Add `osu!(lazer) Updates: June 5, 2025` news post (#13322)
ppy/osu-wiki: news/2025/2025-06-05-osulazer-updates-june-5-2025.md

---
layout: post
title: "osu!(lazer) Updates: June 5, 2025"
date: 2025-06-05 12:00:00 +0000
---

Experts say this might be THE osu!(lazer) update of all time.

![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/banner.jpg)

> You might have noticed that the rate at which news and YouTube videos has slowed down a bit, however the devs have not slowed down one bit, with several smaller updates being pushed out in April. That being said, this update is major, and we're excited to showcase it to you.

Reviewer: ```suggestion
You may have noticed osu!(lazer)-related news and YouTube uploads easing up—but don’t be fooled, the devs haven’t let off the gas for a moment. Alongside several smaller updates being pushed out in April, we're excited to showcase this major update!
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0" allowfullscreen></iframe>
</div>

### Gameplay

> - **The new song select is here for public testing.** To see the new design, simply input the simple *secret code* of... holding the "solo" button for a second before releasing. Very secretive, we know.

Reviewer: a tiny bit of flair wouldn't hurt
```suggestion
- **The new song select is here for public testing!** To see the new design, simply input the simple *secret code* of... holding the "solo" button for a second before releasing. Very secretive, we know 🕵️.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2025-06-05-song-select-v2.mp4" type="video/mp4" preload="none">
    </video>
</div>

> - **The leaderboard is now a skinnable element.** While it's not "skinnable" yet, this means you can manipulate its placement and size via the skin editor under the ruleset "HUD" layer.

Reviewer: ```suggestion
- **The leaderboard is now a skinnable element.** While it's not "fully skinnable" yet, this means you can manipulate its placement and size via the skin editor under the ruleset "HUD" layer.
```

> - **Add non-skinnable position indicator to multiplayer player.** With the update to the leaderboard, [peppy](https://osu.ppy.sh/users/2) wanted to make sure that players are still aware that they are in a multiplayer match, which is why the position indicator is now added to the game. Worry not, you can still hide it with the `Tab` key, but it will always show up during breaks.

Reviewer: ```suggestion
- **Add non-skinnable position indicator to multiplayer player.** With the update to the leaderboard, [peppy](https://osu.ppy.sh/users/2) wanted to make sure that players are still aware that they are in a multiplayer match, which is why the position indicator is now added to the game. Worry not, you can still hide it with the `Tab` key, but it will always show during breaks.
```

also, may be worth showcasing this with https://discord.com/channels/188630481301012481/1191872506420801546/1379884925565796372

- **Mods and freemod can now be used in conjunction with freestyle.** This means that the host can force mods onto the entire lobby, with players still being able to choose to additional mods (for example Hidden on top of forced Double Time for the lobby), and still get to pick any difficulty from the chosen beatmapset.

> - **A new mod has been to osu!taiko.** The Simplified Rhythm mod is exactly what it says on the tin, it turns complicated rhythms such as 1/8 and 1/6 patterns into 1/4, 1/3 rhythms into 1/2, and allows for further modifications via mod settings.

Reviewer: ```suggestion
- **A new mod has been added to osu!taiko.** The Simplified Rhythm mod is exactly what it says on the tin, it turns challenging rhythms such as 1/8 and 1/6 patterns into 1/4 and 1/3 rhythms into 1/2, all based on the player's preference via the mod's settings.
```

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2025-06-05-simplified-rhythm.mp4" type="video/mp4" preload="none">
    </video>
</div>

- **Partial slider body transparency has been added to "argon" skins.** After some player complaints this has been manually added in by peppy, however in the future this will be configurable in-game on a skin-to-skin basis.

![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/argon-transparency.png)

> - **The "Require perfect hits" option has been added for the Mania Perfect mod.** For all the *true* perfectionists out there.

Reviewer: ```suggestion
- **The "Require perfect hits" option has been added for the osu!mania Perfect mod.** For all the *true* perfectionists out there.
```

### Editor

- **The BSS process now has cool new sound effects.** After a complaint was made by a community member, [nekodex](https://osu.ppy.sh/users/102) cooked up brand new satisfying sounds that accompany you through the beatmap submission process.

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2025-06-05-bss-sound-effect.mp4" type="video/mp4" preload="none">
    </video>
</div>

- **Distance snapping can no longer be used to place objects outside of playfield bounds.** The daredevils amongst the mapping community will need to find another way to no-clip out of bounds.
- **Various editor crash fixes and improved logging and errors for BSS.** If you are a fellow nerd, you can check the [full changelog]() for the specific details.

### Other

- **A new "Tachyon" release stream has been added.** This is the equivalent of the "cutting edge" release stream from osu!(stable), but on osu!(lazer). With "Tachyon", you will be able to play on all pre-release versions of the game, and the devs are recommending you give it a try if you have been experiencing stutters on osu!(lazer).

> - **Plays with non-default mod configurations are now clearly displayed.** This means if a user set a score with Double Time, but edited the rate to something other than 1.5x, a new cog icon will appear on the top left of the mod icon. Hovering over the mod icon also gives more specific details.

Reviewer: DT is a bad example for this given it already has its own dedicated display for custom rates. I'd suggest using another example like CL + disabling classic note lock, or HD with the "only fade approach circles" option.

![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/non-default-mod-config.png)

- **You can now join multiplayer/playlist rooms from links.** If a link is shared on either in-game chat or anywhere outside the game, clicking on it now automatically makes you join.

> - **Replay playback issues have mostly been resolved.** If you are someone who's looking through replays on osu!(lazer), you should notice a huge different with this release.

Reviewer: could maybe elaborate a bit on this? video mentions something about rewinding in mania no longer breaking combo.

if not, then apply this

```suggestion
- **Replay playback issues have mostly been resolved.** You'll notice that watching and seeking through replays should feel much better now.
```

- **Plays with non-default mod configurations are now clearly displayed.** This means if a user set a score with the Classic mod, but disabled the "classic note lock" option, a new cog icon will appear on the top left of the mod icon. Hovering over the mod icon also gives more specific details.

![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/non-default-mod-config.png)

- **You can now join multiplayer/playlist rooms from links.** If a link is shared on either in-game chat or anywhere outside the game, clicking on it now automatically makes you join.
- **Replay playback issues have mostly been resolved.** You'll notice that watching and seeking through replays should feel much better now.
- **You can now block users directly within osu!(lazer).** Right-clicking on a user anywhere within the game will now present a "block" option, should you ever need to use it.

---

We know a lot of you held back from trying osu!(lazer) due to not being happy with song select, or because the performance wasn't what you expected. If you belong in that crowd, now is the perfect time to [give it a whirl](https://osu.ppy.sh/home/download) and let the devs know what you think!

> As always, you can refer to the [changelog]() to get the full scoop on all the minor changes that weren't mentioned either on YouTube, or in the news post. See ya next time!

Author: https://github.com/ppy/osu-wiki/pull/13322/commits/efd6627181e29577f2f820fc6ae744a939284688
